src/main/kotlin/org/jitsi/jibri/api/xmpp/JibriIqHelper.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.jitsi.jibri.api.xmpp
+
+import net.java.sip.communicator.impl.protocol.jabber.extensions.jibri.JibriIq
+import net.java.sip.communicator.impl.protocol.jabber.extensions.jibri.JibriStatusPacketExt
+import org.jivesoftware.smack.packet.IQ
+import org.jivesoftware.smack.packet.Presence
+import org.jxmpp.jid.Jid
+
+//TODO: this functionality should be added to JibriIq
+
+/**
+ * Create a result iq from the given [JibriIq]
+ */
+class JibriIqHelper {
+    companion object {
+        fun createResult(jibriIq: JibriIq): JibriIq {
+            val result = JibriIq()
+            result.type = IQ.Type.result
+            result.stanzaId = jibriIq.stanzaId
+            result.to = jibriIq.from
+            return result
+        }
+
+        fun create(from: Jid, type: IQ.Type, status: JibriIq.Status): JibriIq {
+            val jibriIq = JibriIq()
+            jibriIq.to = from
+            jibriIq.type = type
+            jibriIq.status = status
+
+            return jibriIq
+        }
+
+        fun create(from: Jid, type: IQ.Type): JibriIq {
```

Comment:
Why do we have two methods instead of using default parameter values? The type parameter could also default to Type.set (which would allow XmppApi not not override it)

Reply:
yeah i think i was in the mindset of these being moved to `JibriIq` so i was writing them a bit java-style.  i'll fix it.